java/shared/src/main/java/com/squareup/subzero/shared/SubzeroUtils.java

```diff
@@ -130,6 +132,7 @@ protected static List<byte[]> validateAndSort(List<ECKey> pubkeys, byte[] hash,
     // This needs to be the same sort that ScriptBuilder.createRedeemScript does.
     pubkeys.sort(ECKey.PUBKEY_COMPARATOR);
     List<byte[]> sortedSigs = new ArrayList<>();
+    Set<String> seenSigs = new HashSet<>();
```

Comment:
super nit: I think this would work as `Set<ByteString>`, and then you don't have to do the byte string -> byte array -> hex string conversion on line 156 below. Not crucial though.